ArduPlane/RC_Channel.cpp

```diff
@@ -78,6 +78,24 @@ void RC_Channel_Plane::do_aux_function_q_assist_state(AuxSwitchPos ch_flag)
     }
 }
 
+void RC_Channel_Plane::do_aux_function_crow_mode(AuxSwitchPos ch_flag)
+{
+        switch(ch_flag) {
+        case AuxSwitchPos::HIGH:
+            plane.crow_mode = Plane::CrowMode::CROW_DISABLED;
+            gcs().send_text(MAV_SEVERITY_INFO, "CROW Flaps Disabled");
```

Comment:
I'm guessing we could have "Crow" rather than "CROW"?

Reply:
its just style...will do...